pages/index.tsx

```diff
@@ -145,7 +145,7 @@ export default function Home() {
               alt="City from night exposed to attacks."
             />
             <div className="text-white text-2xl absolute bottom-0 left-0 p-4 bg-gradient-to-t from-black to-transparent leading-snug">
-              Cyber-defence to protect privacy and for security
+              Protecting privacy and populations with cyber-defense
```

Comment:
Let's rephrase like the previous one, e.g. : 
Defending data privacy and business secrets...
We can remove cyber-defense, we will make it appear somewhere else

Reply:
why ? I do not see the problem here